docs/user_guide/assessment.rst

```diff
@@ -508,6 +508,18 @@ argument. Here is an example of a pie chart.
 There are many other customizations that can be done. More information can be found in
 :meth:`pandas.DataFrame.plot`.
 
+In order to save a plot, access the `matplotlib` Figure as below and save it with your
+desired filename.
+
+    >>> fig = metric_frame.by_group[["count"]].plot(
+    ...                                         kind="pie",
+    ...                                         subplots=True,
+    ...                                         layout=[1, 1],
+    ...                                         legend=False,
+    ...                                         figsize=[12, 8],
+    ...                                         title="Show count metric in pie chart")
+    >>> fig[0][0].figure.savefig("filename.png")
```

Comment:
I'd recommend checking the way we're doing this for other plots in the same file (basically right above with `literalinclude` and `figure`. This is basically just a reference to an "examples" file from which we copy code and then paste the resulting figure. Your particular code snippet doesn't have a `figure` so that makes it easier. You can add the code you wrote to the corresponding examples file and reference it. The `start-after` and `end-before` tags are quite sensitive, so make sure to add corresponding comments into the examples file that suit your case. You may have to also add an `end-before` for the `literalinclude` that already exists above your section:
```
.. literalinclude:: ../auto_examples/plot_quickstart.py
    :language: python
    :start-after: # Customize plots with kind
    :end-before: # <some new comment you will probably add where your code is pasted in the examples file>
```

I do realize this is quite a bit involved, so feel free to reach out if it doesn't make sense to you. I'm probably not being coherent and can explain everything in more detail!

Reply:
Let me know if this is unclear to you! @rensoostenbach

Reply:
I've already done this right? :) The above piece of code is outdated, but if you check the files as they are right now, I've done it with `literalinclude`.

Reply:
You're right, of course! Let's resolve the comments that are already addressed.